internal/hud/server/controller.go

```diff
@@ -52,7 +50,7 @@ func (s *HeadsUpServerController) isWebsocketConnected() bool {
 }
 
 func (s *HeadsUpServerController) maybeOpenBrowser(st store.RStore) {
-	if s.webURL.Empty() || s.webLoadDone || (bool)(s.noBrowser) {
```

Comment:
this isn't right...noBrowser is always true now, so you need to remove this whole function